var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -70,6 +78,10 @@ class Hwloc(AutotoolsPackage):
     depends_on('libxml2', when='+libxml2')
     depends_on('cairo', when='+cairo')
     depends_on('numactl', when='@:1.11.11 platform=linux')
+    
+    # This does not work with OpenMPI as it does not support the
```

Comment:
Would a more-fundamental reason be that OpenMPI depends on `hwloc` and we can't have cyclic dependencies?

Reply:
Updated text, and added link to the issue.